osbuild/mounts.py

```diff
@@ -126,8 +126,21 @@ def __init__(self, args):
     # pylint: disable=no-self-use
     @abc.abstractmethod
     def translate_options(self, options: Dict) -> List:
+        opts = []
         if options.get("readonly", False):
-            return ["-o", "ro"]
+            opts = ["ro"]
+        if options.get("defaults", False):
+            opts += ["defaults"]
+        if "uid" in options:
```

Comment:
Really really minor thing here but since you're using `options.get` for the boolean arguments I'd prefer it to be used down here as well: `if options.get("uid")`.

Reply:
The thing is, uid can be a valid entry with a value of 0. Wouldn't that prevent the parameter to be added in the list ?

Reply:
I had missed that, it would since it's an actual `int`. Perhaps `options.get("uid", None) is not None` but I'm now perfectly fine with how it already is :)

Reply:
BTW the default return value of `get()` is `None`, so there would be no need to specify it...

Reply:
`options.get()` is how we usually do it, but I think it's fine to keep it as is now with the `in` checks.  Unless you feel strongly about it.